### PR TITLE
adding styles

### PR DESCRIPTION
**Pull request by SummerTimeee**
💯

### CODE REVIEW

File: theme/style.css [hunk 1]
```diff
@@ -3,7 +3,7 @@
   flex-direction: column;
   justify-content: flex-start;
   align-items: center;
-  background-image: url(http://via.placeholder.com/450x150);
+  background-image: url(http://via.placeholder.com/450x150); ;
```
Comment (NAlexPear) [inline theme/style.css]: Extra semicolon!

File: theme/style.css [hunk 2]
```diff
@@ -16,3 +16,7 @@
 }
 
 /* style your grid sections here!  */
+p {
```
Comment (NAlexPear) [inline theme/style.css]: `p` is too broad, please make this selector more specific (e.g. with an ID or an `nth-child` selector)

**Comment by SummerTimeee** [comment]
hey got those changes committed and pushed for ya ;-)

File: theme/style.css (hunk 2 above)
Comment (SummerTimeee) [reply theme/style.css]: hey i fixed it ;-)

Comment (SummerTimeee) [reply theme/style.css]: finalized for sure ;-)

**Pull request closed without merging**